app/models/proxy.rb

```diff
@@ -326,7 +326,7 @@ def save_and_async_deploy(attrs, user)
   end
 
   def async_deploy(user)
-    ProviderProxyDeploymentService.async_deploy(user, self)
+    ProviderProxyDeploymentServiceV1.async_deploy(user, self)
```

Comment:
Sorry. I know I first suggested this name, but can we change it to `ProviderProxyDeploymentV1Service` instead? The way it is now it makes it look like it's the service that is v1, when in fact this is a reference to apicast version.

Reply:
Oki doki